models/demos/wormhole/bert_tiny/tests/test_performance.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import pytest
+import ttnn
+import time
+
+from loguru import logger
+import ttnn
+from ttnn.model_preprocessing import preprocess_model_parameters
+from models.utility_functions import (
+    enable_persistent_kernel_cache,
+    disable_persistent_kernel_cache,
+)
+from models.perf.perf_utils import prep_perf_report
+from models.perf.device_perf_utils import run_device_perf, check_device_perf, prep_device_perf_report
+from transformers import BertForQuestionAnswering
+from models.demos.wormhole.bert_tiny.tt.bert_tiny import bert_for_question_answering
+from models.utility_functions import is_wormhole_b0, skip_for_grayskull
+
+
+def get_expected_times(bert_tiny):
+    return (15.0, 9.2)
+
+
+@skip_for_grayskull()
+@pytest.mark.models_performance_bare_metal
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 24576}], indirect=True)
+@pytest.mark.parametrize("batch_size", [8])
```

Comment:
It looks like when this test is run on n300 the batch-per-device=4 and when it's run on n150 the batch-per-device=8. It would be good to test the same batch-per-device for any number of devices, potentially by treating the batch_size parameter as the per-device batch and multiplying by the number of devices when creating the inputs (and adding more options for batch size if desired). 
Same comment for the tests in demo.py and test_bert_tiny_wh.py.

Reply:
Good point. The reason why we are running the model data-parallel so we can increase the total batch size.

Reply:
Updated the pipeline to run the test for batch-per-device=8 for n300 and n150.